packages/iTwinUI-react/src/core/Table/actionHandlers/selectHandler.ts

```diff
@@ -56,6 +56,109 @@ export const onSelectHandler = <T extends Record<string, unknown>>(
   onSelect?.(selectedData, newState);
 };
 
+/**
+ * Handles selection when clicked on a row while shift is selected.
+ */
+export const onShiftSelectHandler = <T extends Record<string, unknown>>(
+  state: TableState<T>,
+  action: ActionType,
+  instance: TableInstance<T>,
+) => {
+  const resetSelection = () => {
+    instance.toggleAllRowsSelected(false);
+  };
+
+  const getGreaterId = (id: string) => {
```

Comment:
can you add a comment here? it's not obvious what "greter id" means or what this function is doing

Reply:
**[DONE]** Ohh, that's true. I renamed `getGreaterId()` to `getNextSiblingId()` and also added a JSDocs comment with some examples.

Reply:
the jsdoc examples are still using the old `getGreaterId` name

Reply:
**[OUTDATED]** Thanks, luckily with the new code, there are no helpers